azure-pipelines.yml

```diff
@@ -41,15 +41,30 @@ steps:
     dockerHubImageName: $(imageName)
     CC_TEST_REPORTER_ID: $(ccReporterID)
 - script: |
+    set -x
     $DOCKER_OVERRIDE exec -T web /bin/sh -c 'bundle config --local disable_exec_load true'
     $DOCKER_OVERRIDE exec -T web /bin/sh -c 'bundle exec rake parallel:setup'
-    $DOCKER_OVERRIDE exec -T web /bin/sh -c 'bundle exec rake "parallel:spec[,,--format progress --format RspecJunitFormatter --out rspec-results.xml --no-profile]"'
+    $DOCKER_OVERRIDE exec -T web /bin/sh -c 'echo "--no-profile
+    --format documentation
+    --format RspecJunitFormatter
+    --out rspec-output/rspec-output<%= ENV['TEST_ENV_NUMBER'] %>.xml" >> app/.rspec'
+    $DOCKER_OVERRIDE exec -T web /bin/sh -c 'bundle exec rake parallel:spec'
+    test_result=$?
+        docker cp s_web_1:/app/rspec-output .
```

Comment:
`s_web_1` is that intentional?

Reply:
Yep that is currently the name of the container

Reply:
Might be worth re-aligning the lines here, I don't think it needs to be indented one more level.